wsl.md

```diff
@@ -0,0 +1,87 @@
+# Running a local Expo development environment in Windows Subsystem for Linux (WSL)
+You can use Expo tools in Windows via the Powershell terminal, but many developers appreciate being able to develop in a Linux environment via Windows Subsystem for Linux (WSL). WSL 2 enables Windows users to run a full Linux environment alongside their Windows applications. As Expo tools support Linux, they can run inside of WSL. The primary challenge with using Expo tools inside of WSL is that WSL is virtual machine with its own separate IP address internal to your machine, and the Expo CLI runs a development server that you may want to connect to via a development build running on a mobile device on your local network. Read on about how to configure WSL so the Expo CLI's dev server is accessible from your LAN even when running in WSL, just as it would be if you were running Linux or another operating system directly.
+
+## Recommended prerequisites
```

Comment:
```suggestion
## Prerequisites
```

A prerequisite is always something that is required to do before going through the rest of the steps inside a guide so I don't think we need to add "Recommended".

Reply:
What about "Recommended setup?" These requirements aren't hard-and-fast, they're just the conditions under which I happened to get everything working.

Reply:
That sounds good 👍 and yeah I understand the context you want to provide here.